.github/workflows/ci.yml

```diff
@@ -51,9 +51,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install protobuf
+        env:
+          GH_TOKEN: ${{ github.token }}
         run: |
-          curl -OL https://github.com/google/protobuf/releases/download/v25.3/protoc-25.3-linux-x86_64.zip
-          unzip protoc-25.3-linux-x86_64.zip -d $HOME/.local
+          gh release download --repo google/protobuf --pattern '*-linux-x86_64*' --output protoc.zip
```

Comment:
Looks like this version isn't compatible with the "import wrapper" call https://github.com/Shopify/tapioca/actions/runs/8474508328/job/23221014800?pr=1852#step:6:722